test/pizza-guy.spec.js

```diff
@@ -64,4 +79,40 @@ describe('pizza-guy', function() {
       expect(pizzaGuy.deliver([]).start()).toBe(undefined);
     });
   });
+
+  describe('physical files', function() {
+    it('should call onComplete after all the images where downloaded', function(done) {
+      this.timeout(4000);
+
+      const baseUrl = 'http://local.foo.com';
+      const images = [
+        'fixtures/images/image-1.jpg',
+        'fixtures/images/image-2.jpg',
+        'fixtures/images/image-3.jpg'
+      ];
+
+      const imagesUrls = images.map(image => `${baseUrl}/${path.parse(image).base}`);
+
+      mockImages(baseUrl, images);
+
+      pizzaGuy
+        .deliver(imagesUrls)
+        .onAddress('./_tmp/foo')
```

Comment:
Is this the default path where nock is writing or from where is this coming from?